go/proofs.pb.go

```diff
@@ -492,23 +490,21 @@ func _CommitmentProof_OneofSizer(msg proto.Message) (n int) {
 //the original key-value pairs into the basis hash, for many existing
 //merkle trees.
 //
-//key and value are passed in. So that the signature of this operation is:
-//leafOp(key, value) -> output
+//key and valuehash are passed in. So that the signature of this operation is:
+//leafOp(key, valuehash) -> output
 //
-//To process this, first prehash the keys and values if needed (ANY means no hash in this case):
+//To process this, first prehash the keys if needed (ANY means no hash in this case):
 //hkey = prehashKey(key)
-//hvalue = prehashValue(value)
 //
 //Then combine the bytes, and hash it
-//output = hash(prefix || length(hkey) || hkey || length(hvalue) || hvalue)
+//output = hash(prefix || length(hkey) || hkey || valuehash)
 type LeafOp struct {
-	Hash         HashOp   `protobuf:"varint,1,opt,name=hash,proto3,enum=ics23.HashOp" json:"hash,omitempty"`
-	PrehashKey   HashOp   `protobuf:"varint,2,opt,name=prehash_key,json=prehashKey,proto3,enum=ics23.HashOp" json:"prehash_key,omitempty"`
-	PrehashValue HashOp   `protobuf:"varint,3,opt,name=prehash_value,json=prehashValue,proto3,enum=ics23.HashOp" json:"prehash_value,omitempty"`
```

Comment:
Why do you remove this? We actually need this info for validation.
I think you also refer to it on this line: https://github.com/confio/ics23/pull/31/files#diff-9f9b21196d67539dcb8d06f539269585R68

Reply:
You also know editing this file by hand is a no-no and you should edit proofs.proto in root, right?
(I accept it for you doing debuging, but this is needed to be fixed properly before any merge).

That also means we will need to update rust and js code with the new structs and also the generation code....